marshal_test.go

```
@@ -33,6 +33,7 @@ import (
 	"fmt"
 	"math"
 	"math/big"
+	"strconv"
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(goimports)